src/AWS.Deploy.Orchestration/CdkProjectHandler.cs

```diff
@@ -114,7 +114,7 @@ public async Task DeployCdkProject(OrchestratorSession session, CloudApplication
             {
                 // Ensure region is bootstrapped
                 var cdkBootstrap = await _commandLineWrapper.TryRunWithResult($"npx cdk bootstrap aws://{session.AWSAccountId}/{session.AWSRegion} -c {Constants.CloudFormationIdentifier.SETTINGS_PATH_CDK_CONTEXT_PARAMETER}=\"{appSettingsFilePath}\" --template \"{_workspaceMetadata.CDKBootstrapTemplatePath}\"",
-                    workingDirectory: cdkProjectPath,
+                    workingDirectory: _workspaceMetadata.DeployToolWorkspaceDirectoryRoot,
```

Comment:
Should we stop passing in the `appSettingsFilePath` context parameter now that we aren't running in the project directory?

Reply:
Yes. I removed it.